src/modules/compliance/src/lib/CMakeLists.txt

```diff
@@ -33,6 +33,7 @@ add_library(compliancelib STATIC
     Base64.cpp
     Engine.cpp
     Procedure.cpp
+    Indicators.cpp
```

Comment:
nit: can arrange the files in alphabetical order when the list grows more (like here) for easier spotting of contents. We use this in several places in this project. Not required and not that important but worth considering IMO.